scala/scala-impl/test/org/jetbrains/plugins/scala/base/ScalaLightPlatformCodeInsightTestCaseAdapter.scala

```diff
@@ -95,7 +91,7 @@ abstract class ScalaLightPlatformCodeInsightTestCaseAdapter extends LightPlatfor
     disposeLibraries(getModule)
     val allJdks = ProjectJdkTable.getInstance.getAllJdks
     WriteAction.run(() => {
-      def foo() =
+      def foo(): Unit =
```

Comment:
@SrTobi I think this method should be inlined 😄

Reply:
Seems it was an artifact of automatic JavaToScala conversion. I simplified this.